src/Hero.java

```diff
@@ -0,0 +1,83 @@
+import java.util.Random;
+
+public class Hero {
+    Random generator = new Random();
+    public Coordinates position = new Coordinates(0, 0);
+    char symbol;
+
+    public Hero(int height, char[][] matrix) {
+        setInitialPosition(height, matrix);
+        symbol = 'H';
+    }
+
+    private void setInitialPosition(int height, char[][] matrix) {
+        int rowIndex;
+        do  {
+            rowIndex = generator.nextInt(height-1);
```

Comment:
Następnym krokiem będzie zrobienie 3 leveli.
Zastanów się jak to zrobić żeby te level były jakoś zapisane i ładowały się jeden po drugim jak kamień dostanie się do dziury (nowe pole).
Dodaj słowo "reset" które resetuje level.
Level musi mieć zapisaną pozycję gracza, kamienia, wyjścia i ścian (na razie zrób żeby mapa zawsze była kwadratem takim samem, ale plusy mogą też być w środku i być przeszkodami)